model/Image.php

```diff
@@ -564,6 +564,11 @@ public function generateSetSize(Image_Backend $backend, $width, $height) {
 		return $backend->paddedResize($width, $height);
 	}
 
+	/**
+	 * TODO: Documentation.
```

Comment:
Seems like this is the opportunity to write that documentation? :D

You could just copy the docs from the method below `Resize this image for the CMS. Use in templates with $CMSThumbnail.`